src/ts/typings/types.d.ts

```diff
@@ -19,4 +21,13 @@ declare namespace Ltl {
     hidden?: boolean;
     deleted?: boolean;
   }
+
+  interface FeaturePromptContent {
+    id: string;
+    prompt: string; // the prompt message for help
+    icon: string;
+    hasDismissed: Writable<boolean>;
+    neverShow: Writable<boolean>;
```

Comment:
Refer above. `import('svelte/store').Writable` should do the trick.

Reply:
Fixed in e378179a506cf60e94a1a5764354f21ea2fa0b4b